storybookjs/storybook
# Custom scrollbars #5421

Issue: #5421 

## What I did
- Styled scrollbars using [SimpleBar](https://github.com/Grsmto/simplebar) for nicer scrollbar UX, hoverable, customizable, cross-browser, etc. This is the same lib that codepen.io uses.

Note: I initially tried implementing this with CSS-only but was unable to achieve the desired styling and interaction.

## How to test
Custom scrollbars are present in 4 places.  Resize your window or drag to expand the Sidebar to see the overflow behavior. Omission from the preview iframe is intentional.

**Sidebar** 
![image](https://d1sz9tkli0lfjq.cloudfront.net/items/3W2v450m2b1E23430N1X/Screen%20Recording%202019-02-25%20at%2004.36%20PM.gif?v=14fdca3e)

**Canvas Toolbar**
![image](https://d1sz9tkli0lfjq.cloudfront.net/items/023s033s1f330n192b2c/Screen%20Recording%202019-02-26%20at%2002.13%20PM.gif?v=5a6363aa)

**Addon bar**
![image](https://d1sz9tkli0lfjq.cloudfront.net/items/0A3x21181331042P3e23/Screen%20Recording%202019-02-25%20at%2002.56%20PM.gif?v=17322ba2)


**Addon panel**
![image](https://d1sz9tkli0lfjq.cloudfront.net/items/163D3S0W3c2K0o1r1c14/Screen%20Recording%202019-02-26%20at%2002.14%20PM.gif?v=64411607)
